devpack/src/test/java/io/neow3j/devpack/template/ExampleContract.java

```diff
@@ -0,0 +1,23 @@
+package io.neow3j.devpack.template;
+
+import io.neow3j.devpack.framework.SmartContract;
+import io.neow3j.devpack.framework.Storage;
+import io.neow3j.devpack.framework.annotations.EntryPoint;
+import io.neow3j.devpack.framework.annotations.ManifestExtra;
+import io.neow3j.devpack.framework.annotations.ManifestFeature;
+
+@ManifestExtra(key = "author", value = "AxLabs")
```

Comment:
Oh, you have for now key and value... I see.

Reply:
So, it means that we would use either the `ManifestExtra` or the `ManifestFeature`, for the name, description, and author, right?
What if both are specified? Which one will override the other?

Reply:
The extra field in the manifest file could contain any key value pair, therefore I changed the @Manifest annotation and separated it into `ManifestExtra` and `ManifestFeature`. Both represent two different sections in the manifest file, i.e. the `feature` and the `extra` sections.